db/meter_readings.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE meter_readings
 (
-    meter_readings_id int PRIMARY KEY,
+    meter_readings_id SERIAL PRIMARY KEY,
```

Comment:
int GENERATED ALWAYS AS IDENTITY is better: https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-identity-column/

Reply:
Yeah I just investigate about that and you're right, SERIAL is only on PostgreSQL and is older than int GENERATED ALWAYS AS IDENTITY. Thanks.